assets/js/delete-task.js

```diff
@@ -0,0 +1,9 @@
+import { CLICK_EVENT } from "/assets/js/constants.js";
+
+function deleteTaskEvent(deleteButton, task, taskList, allTasks, id) {
```

Comment:
are we deleting a Task `event` or a Task rather? function name doesn't express the implementation properly.

renaming the function to `onDeleteTask`, `deleteTask` or even `deleteTaskHandler` would be good as it expresses the implementation details more appropriately.